easybuild/easyconfigs/y/Yade/Yade-1.20.0-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,55 @@
+easyblock = 'CMakeMake'
+
+name = 'Yade'
+version = '1.20.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://yade-dem.org/doc/'
+description = """ Yade is an extensible open-source framework for discrete numerical models, 
+ focused on Discrete Element Method. The computation parts are written in c++ using flexible object model, 
+ allowing independent implementation of new alogrithms and interfaces. 
+ Python is used for rapid and concise scene construction, simulation control, postprocessing and debugging. """
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['https://launchpad.net/yade/trunk/yade-1.00.0/+download/']
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [
+    ('Loki', '0.1.7'),
+    ('Boost', '1.60.0', versionsuffix),
+    ('GL2PS', '1.3.9', '-Mesa-11.2.1'),
+    ('Python', '2.7.11'),
+    ('IPython', '3.2.3', versionsuffix),
+    ('PyGTS', '0.3.1', versionsuffix),
+    ('Eigen', '3.2.7'),
+    ('minieigen', '0.5.3', versionsuffix),
+    ('Qt', '4.8.7', '-GLib-2.48.0'),
+    ('GTS', '0.7.6'),
+    ('libQGLViewer', '2.6.3', '-Mesa-11.2.1'),
+    ('zlib', '1.2.8'),    
+    ('CGAL', '4.8', versionsuffix),
+]
+
+builddependencies = [('CMake', '3.5.2')]
+
+separate_build_dir = True
+
+configopts = '-DENABLE_GUI=OFF '
+configopts += '-DENABLE_VTK=OFF '
+configopts += '-DENABLE_LINSOLV=OFF '
+configopts += '-DENABLE_PFVFLOW=OFF '
+configopts += '-DCGAL_INCLUDE_DIR=$EBROOTCGAL/include '
+configopts += '-DGMP_INCLUDE_DIR=$EBROOTGMP/include '
+configopts += '-DGMP_LIBRARIES=$EBROOTGMP/lib '
+configopts += '-DGMPXX_LIBRARIES=$EBROOTGMP/lib '
+
+#to disable CGAL uncomment the following line:
```

Comment:
@jrbosch any particular reason you include this? Why would you want to disable CGAL?